video_xblock/static/html/brightcove.html

```diff
@@ -18,7 +18,7 @@
         data-embed="default"
         data-application-id
         data-setup='{"playbackRates": [0.5, 1, 1.5, 2] }'
-        width="829"
+        width="100%"
         height="511"
```

Comment:
Ok, we need it because it's in the iframe.
But I don't like that it looks like a random number from to first sight. Can we make it `100%`, `""`, `1px` or some other value that will tell it's actually not being used?